### PR TITLE
Missing attrs added to toString method

### PR DESCRIPTION
**Pull request by afilardo**
We weren't logging this attrs, so I realized that they were missing.
I'm not sure if in the non-encrypted case, this is intentional, so I have left it as it was

**Description**

**Tested scenarios**

**Fixed issue**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Add this dependency to your project's POM:
 <dependency>
   <groupId>com.adyen</groupId>
   <artifactId>adyen-java-api-library</artifactId>
-  <version>2.4.1</version>
+  <version>2.5.0</version>
```
Comment (AlexandrosMor) [inline README.md]: We increase the version number when merging develop to master. Could you please revert it ?

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <groupId>com.adyen</groupId>
     <artifactId>adyen-java-api-library</artifactId>
     <packaging>jar</packaging>
-    <version>2.4.1</version>
+    <version>2.5.0</version>
```
Comment (AlexandrosMor) [inline pom.xml]: Same here

File: src/main/java/com/adyen/Client.java [hunk 3]
```diff
@@ -41,7 +41,7 @@
     public static final String MARKETPAY_FUND_API_VERSION = "v3";
     public static final String MARKETPAY_NOTIFICATION_API_VERSION = "v1";
     public static final String LIB_NAME = "adyen-java-api-library";
-    public static final String LIB_VERSION = "2.4.1";
+    public static final String LIB_VERSION = "2.5.0";
```
Comment (AlexandrosMor) [inline src/main/java/com/adyen/Client.java]: Same here

**Comment by martinsrenato** [comment]
Hi @afilardo,
Can you rebase your branch to develop?
We had to make some changes there that affected this PR.
Thanks.

**Comment by afilardo** [comment]
Sorry, I don't have the branch anymore

**Comment by martinsrenato** [comment]
@afilardo I copied your changes and opened a new PR with the rebased code from develop. I'll close this one and we will follow from that. You can check on #236.

**Pull request closed without merging**